docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -180,6 +211,14 @@ Removing network nebula-docker-compose_nebula-net
 
 To update the images of the Graph Service, Storage Service, and Meta Service, run `docker-compose pull` in the `nebula-docker-compose` directory.
 
+### `ERROR: toomanyrequests` when `docker-compose pull`
+
+You may meet the following error.
+
+`ERROR: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit`.
+ 
+This is the `rate limits` of `docker.com`. Follow the instructions in [this page](https://www.docker.com/increase-rate-limit).
```

Comment:
You have met the rate limit of Docker Hub. Learn more on [Understanding Docker Hub Rate Limiting](https://www.docker.com/increase-rate-limit).